### ISSUE TITLE
collecting google analytics on mentor training 

### ISSUE THREAD

**Issue by amyiridescent** [Iridescent-CM/technovation-app#1766]
With training google slide deck being embedded, can we collect some information? Such as how long people are staying on the training? And if they go back to the training during the year?

**Comment by joemsak** [comment]
@amyiridescent yes, the google analytics script is already tracking this, you'll want to look at activity for the page referenced by `/mentor-training` https://analytics.google.com/analytics/web/#/report/content-pages/a4055471w105362134p109648134/explorer-table.plotKeys=%5B%5D&_r.drilldown=analytics.pagePath:~2Fmentor-training

**Comment by joemsak** [comment]
Oops, should have said in my original comment, but if that works, please confirm if we can close this card. Thanks!

**Comment by amyiridescent** [comment]
Can I take a day or two to make sure I understand this information being collected on the analytics? I want to make sure I don't need help setting up anything additional. 

Also, I assume it is hard to know which mentor (user ID) is spending how long on the training, but want to make sure that is not a false assumption I am making.

**Comment by Allicolyer** [comment]
Hey @amyiridescent, do you want to continue this conversation?

**Comment by joemsak** [comment]
For now, you can already use google analytics to see how long the average user is staying on that page. I am not so sure that event tracking will work for what you're hoping: tracking individual users. Because event tracking requires an action, such as clicking on something. It's not meant for keeping track of time spent on a page. But I am not a google analytics expert, perhaps someone else knows more than I do. Is there someone on staff who uses google analytics regularly and is more familiar with the toolsets available?

**Comment by stenington** [comment]
This is your best bet. It's a bit complicated, but we have it set up on CM. https://support.google.com/analytics/answer/3123662?hl=en&ref_topic=3123660

**Comment by joemsak** [comment]
Very cool, thank you Mike!

**Comment by Allicolyer** [comment]
@amyiridescent do you have everything you need here? Perhaps we need to show Jess and Mackenzie how to access this information?
